tests/gold_tests/h2/http2.test.py

```diff
@@ -26,7 +26,7 @@
 Test.SkipUnless(
     Condition.HasCurlFeature('http2')
 )
-Test.ContinueOnFail = True
+#Test.ContinueOnFail = True
```

Comment:
I'm not sure why these changes were made, but seems like something we want to have on 9.2 as well. If so, it's unfortunate that we can not simply cherry pick a commit to have these changes.

Reply:
These changes can probably be reverted, actually. Definitely the ContinueOnFail comment was likely for Susan to debug something a while back. Let me revert them and if it causes problems in CI we can circle back around on this.